plugins/destination/duckdb/client/write.go

```diff
@@ -125,12 +127,60 @@ func (c *Client) Write(ctx context.Context, msgs <-chan message.WriteMessage) er
 	return nil
 }
 
+func (c *Client) appendRows(table *schema.Table, msgs message.WriteInserts) error {
+	fmt.Println("appendRows", table.Name)
+	appender, err := duckdb.NewAppenderFromConn(c.conn, "", table.Name)
+	if err != nil {
+		return fmt.Errorf("failed to create appender for %s: %w", table.Name, err)
+	}
+
+	for _, msg := range msgs {
+		arr := transformRecordToGoType(msg.Record)
+		for i := range arr {
+			b, _ := json.Marshal(arr[i])
```

Comment:
err shouldn't be ignored IMO

Reply:
it's just for `fmt.Println("appendRows", string(b))`